docs/src/main/tut/docs/types.md

```diff
@@ -16,6 +16,8 @@ title: "Supported types"
 | `scala.Option[T]                       `  | ✓              |✓                     |✓|
 | `scala.Boolean                         `  | ✓              |✓                     |✓`*`|
 | `scala.Array[Byte]                     `  | ✓              |✓                     |✓|
+| `scala.List[T: ValueMapper]            `  | ✓              |                      |✓|
+| `scala.Set[T: ValueMapper]             `  | ✓              |                      |✓|
```

Comment:
I am unsure if a `List[T]` or a `Set[T]` can be a **property**, and as such be a field of a **case class*.

See this: https://neo4j.com/docs/cypher-manual/current/syntax/values/

At first, they say that it is possible, for simple types. But latter they say it wont work on general _(maybe they mean that lists of lists are not valid?)_.

Maybe, you should add the ✔️ but with another _mark_ with that link?

Reply:
As far as I remember, it's not possible at the moment. I expect that library users will submit a ticket if they find a way to return it 😁